docs/en/basechain-upgrade-new-api-for-interacting-with-transfer-gateway.md

```diff
@@ -1,41 +1,35 @@
 ---
-id: how-to-migrate-from-plasmachain-to-basechain
-title: How to Migrate your DApp from Plasmachain to Basechain
+id: basechain-upgrade-new-api-for-interacting-with-transfer-gateway
+title: Basechain Upgrade- The New API for Interacting with the Transfer Gateway
 ---
 
 ## Purpose
 
-This tutorial walks you through the process of migrating your DApp from Plasmachain to Basechain.
-
-Please note that that we are set to roll out Basechain towards the end of November 2019. This upgrade introduces a few **breaking changes** to how assets are transferred from Loom to Ethereum.
-
-Follow the next steps to make sure your DApp works once Basechain gets rolled out.
-
-> Once you've updated your code, we encourage you to test it against `extdev` to rule out any potential issues.
+As part of the Basechain upgrade, we'll be updating the Transfer Gateway. To make the transition as smooth as possible, we've added a new API to `loom-js` for interacting with the Transfer Gateway. This page outlines the changes you'll need to make in order to ensure your DApp continues to work after the Transfer Gateway update goes live.
 
 ## 1. Update the Ethereum Transfer Gateway address
 
-As part of the upgrade, we're updating the transfer gateway smart contracts deployed on the Rinkeby testnet and Ethereum main net. If these are hardcoded in your codebase, make sure you use the addresses listed here:
+We're updating the transfer gateway smart contracts deployed on the Rinkeby testnet and Ethereum main net. If these are hardcoded in your codebase, make sure you use the addresses listed here:
```

Comment:
We're not updating anything on Rinkeby, it's has already been updated, so don't need to mention it here.